bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/nici/Test.java

```diff
@@ -73,8 +74,12 @@ public static void main(String[] argv) throws IOException {
                     new ObservingConditionParameters(2, 2, 4, 3, 2.3);
             
             NiciParameters np =
-                    new NiciParameters("R",
-                    NiciParameters.NDA);
+                    new NiciParameters(
+                        "H",                    // channel1
+                        "ndfilt_clear",         // channel2
+                        "80",                   // pupil
+                        "",                     // instrument mode, unused?
+                        "h5050");               // dichroic position
```

Comment:
Is the idea that stuff like this will be based on the program model (e.g., `edu.gemini.spModel.gemini.nici.NICIParams`) when you're done refactoring?  So you'd pass in `DichroicWheel.H5050_BEAMSPLITTER` there instead of the `String` "h5050"?